packages/datadog-plugin-cypress/test/index.spec.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+const getPort = require('get-port')
+const { expect } = require('chai')
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+const appServer = require('./app/app-server')
+const { ORIGIN_KEY } = require('../../dd-trace/src/constants')
+const {
+  TEST_FRAMEWORK,
+  TEST_TYPE,
+  TEST_NAME,
+  TEST_SUITE,
+  TEST_STATUS,
+  CI_APP_ORIGIN
+} = require('../../dd-trace/src/plugins/util/test')
+
+describe('Plugin', () => {
+  let cypressExecutable
+  let appPort
+  let agentListenPort
+  withVersions(plugin, ['cypress'], (version, moduleName) => {
+    beforeEach(() => {
+      return agent.load(['cypress']).then((agentPort) => {
+        agentListenPort = agentPort
+        cypressExecutable = require(`../../../versions/cypress@${version}`).get()
+        return getPort().then(port => {
+          appPort = port
+          appServer.listen(appPort)
+        })
+      })
+    })
+    afterEach(() => {
+      return Promise.all([
```

Comment:
This would be a bit easier to read as 2 blocks, for example `afterEach(() => agent.close())` and `afterEach(done => appServer.close(done))`.

Reply:
yeah, that's right 👍